web/styles/portico/footer.css

```diff
@@ -19,7 +19,7 @@
 
     .footer__container {
         max-width: 1132px;
-        padding: 52px 52px 0;
+        padding: 24px 52px 52px;
```

Comment:
Can we do a symmetric 24px as the bottom margin? I'll post screenshots in Zulip for what it looks like.